lib/elixir/test/elixir/kernel/overridable_test.exs

```diff
@@ -92,6 +92,29 @@ defmodule Kernel.Overridable do
   end
 end
 
+defmodule Kernel.ExampleBehaviour do
+  @callback required_callback :: any
+  @optional_callbacks optional_callback: 0
```

Comment:
Apparently Erlang [does not raise any error](http://stackoverflow.com/documentation/erlang/7004/behaviours/25643/optional-callbacks-in-a-custom-behaviour) if you specify a callback in `optional_callbacks` that hasn't been defined.

Perhaps we should emit a warning ourselves in this case?